lib/raven/interfaces/http.rb

```diff
@@ -15,6 +15,7 @@ class HttpInterface < Interface
     def initialize(*arguments)
       self.headers = {}
       self.env = {}
+      self.cookies = nil
       super(*arguments)
```

Comment:
I remember now - without this, cookies is not included in `instance_variables`, so it's not included in the to_hash method (unless you set it to something yourself). [That caused this test to fail](https://github.com/getsentry/raven-ruby/pull/201/files#diff-2e67fc5726ac8c5cab5523f71896e49cR175).